app/src/main/java/ru/prometeydev/movie/data/adapters/ActorsAdapter.kt

```diff
@@ -0,0 +1,63 @@
+package ru.prometeydev.movie.data.adapters
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import ru.prometeydev.movie.R
+import ru.prometeydev.movie.data.models.Actor
+
+/**
+ * Адаптер для списка актеров
+ */
+class ActorsAdapter : RecyclerView.Adapter<ActorsAdapter.ActorsViewHolder>() {
+
+    private var actors = listOf<Actor>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ActorsViewHolder {
+        val itemView = LayoutInflater.from(parent.context)
+            .inflate(R.layout.view_holder_actor, parent, false)
+
+        // Вычисляем ширину разделителя между элементами списка
+        val dividerWidth = (parent.context.resources.getDimension(R.dimen.margin_small) * parent.context.resources.displayMetrics.density).toInt()
+        // Устанавливаем ширину элемента списка
+        itemView.layoutParams.width =  (parent.measuredWidth - dividerWidth) / NUMBERS_OF_ITEM_TO_DISPLAY
```

Comment:
Aha, I'm not sure how to improve here. Intuition says that this code is hard. Probably what I mention about the just have margin in the item is easier to understand.

Reply:
I understand that I myself did not achieve the result on the first attempt, but I really do not know how it could have been made easier. Basically, there was information, to calculate the size myself, which I did.

Reply:
Interesting, from my experience in projects we don't resize much due to landscape. As I told we treat the landscape as just one weird device with crazy screen size. With this approach, I would just keep actors size same fixed for both orientations.

Reply:
Ok, I thought about it. I was just embarrassed by the GIF to the activity before last homework. There in the gif, a variant of displaying the orientation in landcape was demonstrated, 4 actors in a row, while the pictures were stretched. But I think you can leave the dimensions static, here you are right.